_env/README.md

```diff
@@ -1,67 +1,63 @@
-# Deployment
+# Deploying the Website
 
 ## Setting up
 
-To setup, you need to have the following installed:
+This assumes you've got access to a kubernetes cluster. We have [a guide][guide]
+on how we do this at Student Robotics.
 
-* [GCloud][gcloud]
+### Deploy the application for the first time
```

Comment:
Not sure this being h3 makes sense (given the previous heading is h2 and this isn't really a sub-section)?

Reply:
I fixed this up in a new commit.

Reply:
https://github.com/srobo/website/pull/155/commits/b4da306d387d209967dbc3f77407b0f4d3b13d23